Add v2.3 Release note (#384)
risingwavelabs/risingwave-docs: docs/sql/functions-operators/sql-function-datetime.md

| Operation | Description | Example |
| ----------- | ----------- | ----------- |

> | *timestamp_value* AT TIME ZONE *zone* → *timestamptz_value* | Converts times between timestamp and timestamptz. Invalid local time during daylight saving forward is not supported. Ambiguous local time during daylight saving backward is interpreted as after the transition. | `'2021-12-31 16:00:00'::timestamp AT TIME ZONE 'us/pacific'` → `2022-01-01 00:00:00+00:00` |

Reviewer: ```suggestion
| *timestamp_value* AT TIME ZONE *zone* → *timestamptz_value* | Converts times from timestamp into timestamptz. Invalid local time during daylight saving forward is not supported. Ambiguous local time during daylight saving backward is interpreted as after the transition. | `'2021-12-31 16:00:00'::timestamp AT TIME ZONE 'us/pacific'` → `2022-01-01 00:00:00+00:00` |
```

There is one more form that converts times from timestamptz into timestamp, which we should also add in this PR. `'2022-01-01 00:00:00Z'::timestamp with time zone AT TIME ZONE 'US/Pacific'` -> `2021-12-31 16:00:00`

Reviewer 2: @xiangjinwu Thanks for the feedback! I've explicitly added the timestamptz into timestamp operation and example to the latest commit.
